mypy/nodes.py

```diff
@@ -2755,6 +2755,7 @@ def __init__(self, target: 'mypy.types.Type', fullname: str, line: int, column:
         self.alias_tvars = alias_tvars
         self.no_args = no_args
         self.normalized = normalized
+        self._is_recursive = None  # type: Optional[bool]
```

Comment:
Add comment (mention that if non-`None`, this the cached value). Maybe also mention that this is manipulated by `TypeAliasType`, as otherwise a reader may be confused as not much is done here with this.